partner_product_price/data/ir_actions_server.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="action_product_price_list_report" model="ir.actions.server">
+        <field name="name">Products</field>
```

Comment:
should be a more explicit name, don't you think ? something like 'See product prices"

Reply:
Done.